app/Http/Resources/OrganizationResource.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Request;
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class OrganizationResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @return array<string, mixed>
+     */
+    public function toArray(Request $request): array
+    {
+        return ['id' => $this->id,
+            'name' => $this->name,
+            'type' => $this->type,
+            'status' => $this->status,
+            'expertises_area' => IdAndNameResource::collection($this->expertises),
+            'risc_category' => IdAndNameResource::collection($this->riskCategories),
+            'action_type' => IdAndNameResource::collection($this->resourceTypes),
+            'activity_area' => $this->area,
+            'county' => IdAndNameResource::collection($this->activityCounties),
```

Comment:
```suggestion
            'county' => $this->county->name,
            'activity_counties' => $this->activityCounties->pluck('name'),
```

pe lângă județele atașate ariei de activitate, mai trebuie luat și județul sediului. pentru județe am stabilit că ne trebuie doar numele